tensorflow/core/common_runtime/dml/dml_device_factory.cc

```diff
@@ -73,11 +73,11 @@ static bool IsUmaAdapter(const DmlAdapter& adapter) {
   DML_CHECK_SUCCEEDED(D3D12CreateDevice(adapter.Impl()->Get(), feature_level,
```

Comment:
Let's remove the DML_CHECK_SUCCEEDED call when creating the D3D device as well. It really *shouldn't* fail here (we've already created an adapter), but I see no reason to create the risk in the future.